Assets/MRTK/SDK/Editor/Inspectors/Utilities/Solvers/TapToPlaceInspector.cs

```diff
@@ -28,20 +28,13 @@ public class TapToPlaceInspector : UnityEditor.Editor
         private SerializedProperty onPlacingStopped;
 
         // Advanced properties
-        private SerializedProperty updateLinkedTransformProperty;
-        private SerializedProperty moveLerpTimeProperty;
-        private SerializedProperty rotateLerpTimeProperty;
-        private SerializedProperty scaleLerpTimeProperty;
-        private SerializedProperty maintainScaleProperty;
```

Comment:
I think this PR removes a few too many properties. As far as I can tell, `maintainScale` is the only one that doesn't exist in this script or it's base class. Many of them are in the Solver base class, and we want those to continue to be exposed here: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/main/Assets/MRTK/SDK/Features/Utilities/Solvers/Solver.cs

Reply:
Oh actually! The field was renamed to `maintainScaleOnInitialization`. So we don't need to remove it at all! Just update the string reference

Reply:
I will fix it. Thanks

Reply:
Just committed the fix. I've overlooked the properties of the base class. Thanks for the good review.

Reply:
Thank _you_ for contributing this fix!